README.md

```diff
@@ -182,4 +180,4 @@ This project is inspired and helped by the following projects:
 * [adamyy/checkerframework-lsp](https://github.com/adamyy/checkerframework-lsp)
 * [georgewfraser/vscode-javac](https://github.com/georgewfraser/vscode-javac)
 * [adamvoss/vscode-languageserver-java-example](https://github.com/adamvoss/vscode-languageserver-java-example)
-* [DafnyVSCode/Dafny-VSCode](https://github.com/DafnyVSCode/Dafny-VSCode)
+* [DafnyVSCode/Dafny-VSCode](https://github.com/DafnyVSCode/Dafny-VSCode)
```

Comment:
What changed here?

Reply:
This is also a result of my formatting; however, nothing actually changed in this line content-wise; sorry for the confusion.